exercises/connect/connect_test.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'connect'
+
+class ConnectTest < Minitest::Test
+  EMPTY = [
+    '. . . . .',
+    ' . . . . .',
+    '  . . . . .',
+    '   . . . . .',
+    '    . . . . .'
+  ].freeze
+
+  RECTANGLE_BLACK = [
+    '. O . .',
+    ' O X X X',
+    '  O X O .',
+    '   X X O X',
+    '    . O X .'
+  ].freeze
```

Comment:
The [connect.md file](https://github.com/exercism/x-common/blob/master/connect.md) says:
>  (with spaces added for readability, which won't be in the representation passed to your code)

So you either need to move the spaces outside the quotes or run it through a function to strip the spaces before constructing the `Board`s.